out/outfakes/fake_lock_handler.go

```diff
@@ -1,45 +1,19 @@
 // Code generated by counterfeiter. DO NOT EDIT.
-package fakes
+package outfakes
 
 import (
-	"sync"
+	sync "sync"
```

Comment:
Is this change (line 5) needed ?

Reply:
Ah, this file is generated by counterfeiter ?

Reply:
Is the rename of package needed ? We should try to keep the changes with upstream as few as possible